api/converters.py

```diff
@@ -236,7 +236,9 @@ def _prep_stage_gate_info(
   ship_type = STAGE_TYPES_SHIPPING[fe.feature_type]
   rollout_type = STAGE_TYPES_ROLLOUT[fe.feature_type]
 
-  stages = Stage.query(Stage.feature_id == d['id'])
+  # Get all stages associated with the feature, sorted by stage type.
+  stages = Stage.query(Stage.feature_id == d['id']).order(Stage.stage_type)
```

Comment:
This may require a new index.

Reply:
What is the best way to determining this? To push it to staging and see if it complains about indexes? I'm not certain the protocol 😅

Reply:
Yeah, running on staging is probably the best way.   Based on the query added here, I could guess the needed index, but it is better to have NDB tell us.